### ISSUE TITLE
 code behaves baddly when i set weight-decay

### ISSUE THREAD

**Issue by llf1234** [Joker316701882/Additive-Margin-Softmax#16]
hello? when i set weight-decay , No matter what parameter , learning rate and optimizer I set, it will behave baddly ? could you give me some tutor

**Comment by Joker316701882** [comment]
@llf1234 Hey, I face exactly same problem with you. Recently I haven't got enough time to deal with it. Hope you can do some dig about this code. : )

**Comment by Joker316701882** [comment]
@llf1234 
Try it again. I fix the bugs about weight decay on last fc layer.
